package_firmware.py

```diff
@@ -105,7 +105,9 @@ def get_git_revision_short_hash() -> str:
 package_dict["X12_PRO24"] = [['x12_pro24', default_name]]
 package_dict["X12_PRO30"] = [['x12_pro30', default_name]]
 package_dict["Thor300"] = [['Thor300_20s', default_name]]
-package_dict["JetFleetF6"] = [['JetFleetF6', default_name]]
+package_dict["JetFleetF6"] = [['JetFleetF6_20s', default_name],
+                    ['JetFleetF6_24s', default_name],
+                    ['JetFleetF6_32s', default_name]]
```

Comment:
This does not look right to me. For the firmware matching in vesc tool to work each of these should be on its own line. You only combine them like this if you want to have multiple firmware options for the same hardware name. Maybe I don't fully understand the intention here...

Reply:
It's just that the HW is pretty much the same besides the FETs supporting different max voltages, they are all JetFleet F6, so I thought I would called them all JetFleetF6 and then just choose the variant in the submenu 20s, 24s or 32s.

But if it makes more sense to follow having a complete separate entry in the dictionary for each variant, let me know and I will update it.

Reply:
I just made the change to split it back to 3 entries following your recommendation